.changeset/gentle-ads-hang.md

```diff
@@ -0,0 +1,8 @@
+---
+"@aws-amplify/ui-react": patch
+"@aws-amplify/ui": patch
+"@aws-amplify/ui-vue": patch
+"@aws-amplify/ui-angular": patch
```

Comment:
I forget, are we supposed to include changesets in the docs release?

Reply:
Hmm, I think we should wait for #2543 before we create docs release PR.

Reply:
Good catch!